baseplate/experiments/__init__.py

```diff
@@ -172,54 +194,65 @@ def variant(self, name, user=None, bucketing_event_override=None,
         if do_log:
             assert bucketing_id
             self._log_bucketing_event(experiment, variant, user,
-                                      extra_event_fields)
+                                      inputs)
             self._already_bucketed.add(bucketing_id)
 
         return variant
 
-    def _log_bucketing_event(self, experiment, variant, user,
-                             extra_event_fields):
-        if not self._event_queue:
-            return
-
-        extra_event_fields = extra_event_fields or {}
+    def _log_bucketing_event(self, experiment, variant, user, extra_event_fields):
+
+        """ Logs a v2 bucketing event.
+
+            :param dict experiment: An experiment config dict.
```

Comment:
style nit: this all looks oddly indented. we usually just align with the open quote.